health-services/project-factory/src/server/api/microplanApis.ts

```
@@ -0,0 +1,124 @@
+
+/**
+ * Searches for plans based on configuration, tenant, and boundaries.
+ * @param planConfigId The unique identifier for the plan configuration.
+ * @param tenantId The tenant identifier for which the search is performed.
+ * @param boundaries The jurisdiction or boundary information for the search.
+ * @returns The response containing plan details for the specified criteria.
+ */
+export const searchPlan = async (
+  planConfigId: string,
+  tenantId: string,
+  limit:number=1
```

Biome

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)